docs/dev/style-guide.rst

```diff
@@ -43,6 +43,10 @@ Content
   use the ``doc`` role and not a hyperlink.
 * If you are cross-referencing to a section within our website,
   use the ``ref`` role with the label from the `autosectionlabel extension <http://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html>`__.
+* Use ``<abstract concept>`` and ``<variable>`` as placeholders in code and URLs. For instance:
```

Comment:
I was thinking `$variable` to differentiate them?

Reply:
Yes! So on that note, would you prefer ``$variable`` and that the example is ``https://$slug.readthedocs.io`` ?

Reply:
Also, the bash syntax is nice as it give variable boundaries:

`https://${slug}.readthedocs.io`

Reply:
I think that I prefer just one notation and that it's `<variable name>` and `<abstract concept>`. It's because I have a hard time coming up with examples from our documentation of referenced variable names that are 100% bound and not actually also an abstract concept that we introduced in the docs.

So even if we say `https://$slug.readthedocs.io` or `https://${slug}.readthedocs.io`, there actually doesn't exist a real variable called `slug` but just a concept from the text. So this might as well be `<slug>` and no distinction.

Reply:
I think that seems reasonable, especially since we're using `$` variables to actually be explicit vars in places in the application (eg;. `$rest` in redirects). So probably better to preserve them for that.